### DISCUSSION TITLE
Output to Web Browser in Blazor App

### DISCUSSION THREAD

**Discussion by TronActive** [spectreconsole/spectre.console#1193] [Q&A] [answered]
I see in the example documentation https://spectreconsole.net/live/progress you are showing output progress to the web browser. I am building a blazor app that I would like to output some long-running tasks progress just like how Azure DevOps Pipelines has the console/terminal window during the build and release pipeline. Is something like that possible with Spectre.console and Blazor Server?

**Comment by patriksvensson** [comment, marked as answer]
@joe704la What you're seeing in the documentation is an [Asciinema](https://asciinema.org/) recording of actual terminal output. We do not support Blazor in Spectre.Console, only terminals.
